fileupload.go

```diff
@@ -14,7 +14,15 @@ import (
 type FileUploadParams struct {
 	Params
 	Purpose FileUploadPurpose
-	File    *os.File
+
+	// FileData is a reader with the contents of the file that should be uploaded.
+	// Filename is just the name of the file without path information.
```

Comment:
Thanks for the documentation! would you mind just moving the `Filename` doc string down a line so that it's directly over the field that it's documenting?